xml/deployment_scenarios.xml

```diff
@@ -478,9 +508,9 @@
      </step>
      <step>
       <para>
-       Transfer the contents of <filename>/charts/</filename> to the webserver
-       and unpack it to the desired location. The <filename>index.yaml</filename> file
-       must be in the root directory for your webserver.
+       Transfer the contents of <filename>/charts/</filename> to the to the
+       desired location on the webserver . The <filename>index.yaml</filename>
+       file must be in the root directory for your webserver.
```

Comment:
You can drop the `The <filename>index.yaml</filename> file must be in the root directory for your webserver.` You can have your helm cart repository running under a prefix (eg: `http://web.airgapped.lan/suse-charts/`).